src/components/PopupCardDeleteVerify.js

```diff
@@ -0,0 +1,39 @@
+import Popup from "./Popup.js";
+
+export default class PopupCardDeleteVerify extends Popup {
```

Comment:
This class should be named `PopupWithConfirmation`. `PopupWithConfirmation` is a general class. It can be used for any confirmation, not for only card delete confirmation, so there should not be mentions of `card`